chia/util/config.py

```diff
@@ -325,3 +327,17 @@ def load_defaults_for_missing_services(config: Dict[str, Any], config_name: str)
                     defaulted[service]["selected_network"] = "".join(to_be_referenced)
 
     return defaulted
+
+
+PEER_INFO_MAPPING: Dict[NodeType, str] = {
+    NodeType.FULL_NODE: "full_node_peer",
+    NodeType.FARMER: "farmer_peer",
+}
+
+
+def get_unresolved_peer_infos(service_config: Dict[str, Any], peer_type: NodeType) -> List[UnresolvedPeerInfo]:
```

Comment:
Related to the other comment, it seems like we want the result as a set.  Should we just do that here instead of a list?

Reply:
Yes 👍